sympy/functions/elementary/trigonometric.py

```diff
@@ -2405,6 +2453,16 @@ def eval(cls, arg):
         if i_coeff is not None:
             return -S.ImaginaryUnit * acoth(i_coeff)
 
+        if isinstance(arg, cot):
+            ang = arg.args[0]
+            if ang.is_comparable:
+                return ang % pi # restrict to [0,pi)
```

Comment:
This part also needs to be revised, considering that acot takes values in (-pi/2, pi/2] instead of [0, pi).

Reply:
Okay, so basically the revision in `acot/cot` would be exactly the same as `atan/tan`.
And, this will need `atan/cot` and `acot/tan` to be revised for both to lie in range `(-pi/2,pi/2]` and `(-pi/2,pi/2)` respectively, right?